core/common/src/main/java/alluxio/util/io/FileUtils.java

```diff
@@ -295,7 +292,11 @@ public static void createFile(String filePath) throws IOException {
    * @param path path of the directory to create
    */
   public static void createDir(String path) throws IOException {
-    new File(path).mkdirs();
+    try {
+      Files.createDirectories(Paths.get(path));
+    } catch (AccessDeniedException e) {
```

Comment:
same here, why we need to convert `AccessDeniedException` to `IOException`? it is already a subclass of `IOException`